packages/foreman/nodejs-jed/nodejs-jed.spec

```diff
@@ -1,17 +1,24 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
```

Comment:
You're dropping this from package_manifest.yaml so does that mean we can drop the package?

Reply:
This was double included in the package_manifest, I removed the duplicate instance.